opentimelineio/plugins/manifest.py

```diff
@@ -143,6 +145,19 @@ def load_manifest():
     except ImportError:
         pass
 
+    # Discover setuptools-based plugins
+    for plugin in pkg_resources.iter_entry_points("opentimelineio.plugins"):
+        plugin_name = plugin.name
+        try:
+            plugin_entry_point = plugin.load()
+            plugin_manifest = plugin_entry_point.plugin_manifest()
+        except Exception:
```

Comment:
Is this a blank exception because we're not sure what kinds of exceptions could get thrown out of the other plugin's plugin loading?

Reply:
That was the thought. I didn't put too much thought into this behavior, I just implemented it this way to match the contrib adapter handling above.

That said, since people have a choice about the setuptools plugins (unlike the contrib ones) maybe it makes sense to have them hard  fail if there is an issue.